expyfun/io/tests/test_parse.py

```diff
@@ -45,3 +46,49 @@ def test_parse():
     data = read_tab(ec.data_fname, group_end=None)
     assert_equal(len(data[0]['misc']), 2)  # includes between-trials stuff
     assert_equal(len(data[1]['misc']), 2)
+
+def test_reconstruct():
+    """Test Tracker objects reconstruction"""
+    from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
+    from expyfun.io._parse import reconstruct_tracker, reconstruct_dealer
```

Comment:
modify `expyfun/io/__init__.py` to import `reconstruct_tracker` and `reconstruct_dealer`, then you can just do `from expyfun.io import reconstruct_tracker, reconstruct_dealer`

Also don't nest these imports, put them at the top